src/ckeditorinspector.js

```diff
@@ -294,6 +294,8 @@ export default class CKEditorInspector {
 CKEditorInspector._editors = new Map();
 CKEditorInspector._wrapper = null;
 
+export { MiniCKEditorInspector };
```

Comment:
Why does it need to be exported in `ckeditorinspector.js` module? Wouldn't this mean this code will bloat the standard inspector instances?

Reply:
Can we use multiple webpack entries instead https://webpack.js.org/concepts/entry-points/?

Reply:
Done